src/Animation.tsx

```diff
@@ -97,13 +108,17 @@ export default class Animation {
             // geometry.setAttribute('position', new THREE.BufferAttribute(vertices, 3));
 
             // Create a cylinder geometry
-            var cylinderGeometry = new THREE.CylinderGeometry(0.005, 0, 10, 32);
-            const material = new THREE.MeshBasicMaterial({ color: 0x606060 }); 
+            var cylinderGeometry = new THREE.CylinderGeometry(0.005, 0, 20, 32);
+            const material = new THREE.MeshPhongMaterial({ color: 0x0000ff, emissive: 0x0000ff, emissiveIntensity: 0.1 }); 
             const cylinder = new THREE.Mesh( cylinderGeometry, material );
 
             // Apply a random rotation to each line
             cylinder.applyMatrix4(this.getRandomRotationMatrix());
 
+            var angle = (i / this.numLines) * Math.PI * 2;
+            cylinder.position.x = Math.cos(angle) * 0.00001;
+            cylinder.position.y = Math.sin(angle) * 0.00001;
```

Comment:
Why was this added? `cylinder.applyMatrix4(this.getRandomRotationMatrix());` already randomly generated an XYZ direction.

Reply:
I realized that this was actually setting its position. Why though?

Reply:
Because if it's at 0, 0, 0. The light doesn't affect it